config/owasp/suppressions.xml

```diff
@@ -29,5 +29,9 @@
     <notes>jackson.core/jackson-databind@2.15.0</notes>
     <cve>CVE-2023-35116</cve>
   </suppress>
+  <suppress>
+    <notes>bcprov-jdk18on-1.73.jar Bouncy Castle Crypto package</notes>
+    <cve>CVE-2023-33201</cve>
+  </suppress>
```

Comment:
Have you tried updating to 1.76? Apparently this version fixes this CVE

Reply:
add a dependency to 1.76 and removed the CVE suppression.